setup.py

```diff
@@ -779,26 +779,6 @@ def get_provider_package_from_package_id(package_id: str):
     return f"apache-airflow-providers-{package_suffix}"
 
 
-class AirflowDistribution(Distribution):
```

Comment:
We do not need it any more, because with recent setup.py refactor the preinstalled providers were moved back to providers.